app/src/main/res/values/strings.xml

```diff
@@ -700,4 +700,5 @@
     <string name="recent">Recent</string>
     <string name="chapters">Chapters</string>
     <string name="no_app_to_open_intent">No app on your device can open this</string>
+    <string name="clear_queue">Clear queue</string>
```

Comment:
```suggestion
    <string name="clear_queue">Clear Queue</string>
```
Just a suggestion to maintain consistency throughout the app, and overall a cosmetic standard.

Reply:
done